deployment/clouddeploy/gke-workers/base/exporter.yaml

```diff
@@ -32,7 +32,7 @@ spec:
                 cpu: "6"
```

Comment:
should we also bump requests to something like 25? to avoid starving the node of resources. 

IIRC we can't put 32 here because there are required pods/overhead on these nodes which make the total available CPUs always less than what the machine actualy has.

Reply:
Agreed, 32 will never be feasible (it;'s 31.something based on what is being reported in the dashboard), but it's the limit, not the request.